server/lib/packets/packet.go

```diff
@@ -71,6 +72,16 @@ var handlers = map[proto.Payload_Type]func (pkt *RecvPacket, t Transport) {
 			go ProcessChannelList(pkt.From, pkt.Payload.Msgid, pkt.Payload.ChannelListRequest, t)
 		}
 	},
+	proto.Payload_TopicCreateRequest: func (pkt *RecvPacket, t Transport) {
+		if pkt.Payload.TopicCreateRequest != nil {
+			go ProcessTopicCreate(pkt.From, pkt.Payload.Msgid, pkt.Payload.TopicCreateRequest, t)
```

Comment:
__[gometalinter]__
* [gotype] invalid operation: pkt.Payload (variable of type *invalid type) has no field or method Msgid
* [gotype] invalid operation: pkt.Payload (variable of type *invalid type) has no field or method TopicCreateRequest